lib/config.go

```diff
@@ -507,7 +507,7 @@ func (c *NetworkConfig) Validate(onExecution bool) error {
 		}
 
 		for _, pluginDir := range c.PluginDir {
-			if _, err := os.Stat(pluginDir); err != nil {
+			if err := os.MkdirAll(pluginDir, 0755); err != nil {
```

Comment:
should we only create it if the pluginDir doesn't exist?

Reply:
I would keep it: If `pluginDir` is already a directory, `MkdirAll()` does nothing and returns `nil`. If it’s a file, it errors which we had not before. :)